packages/web/src/components/spa/HackAUBG/HackAUBG.jsx

```diff
@@ -20,6 +20,7 @@ export const HackAUBG = () => {
     makeBodyScrollable();
 
     document.body.className = 'hackaubg-container';
+    document.body.style.backgroundColor = '#222222';
```

Comment:
Since probably this value isn't overwritten when we hop back to the main page, a better approach would be to bind this logic to the function which changes the favicons. 

You could find the function in Global.js. Essentially it's an event listener which checks whether the url contains hackaubg or not. Copy this line of code underneath the conditional statement for the former and set a background Color for the main page as well.